CHANGELOG.rst

```diff
@@ -6,7 +6,7 @@ dev
 
 **API Changes (Backward Incompatible)**
 
--
+- Added type hints.
```

Comment:
Are there actually backward incompatible changes?
The next major release might be a long time away...

Reply:
Woops! No these aren't breaking changes, I read this as "Backwards compatible", doh.